File: kung_fury.py
from random import choice
import string
from sys import stdout
from time import sleep

if __name__ == '__main__':
    try:
        with open('script.txt', 'r') as f:
            energy_converter_text = f.read()
    except IOError:
        print('Could not open file')
        exit()

    for i in range(2):
        for char in energy_converter_text:
            stdout.write(char)
            stdout.flush() # Needed to print 1 at a time
            sleep(0.02)

    count = 0
    chars = string.letters + string.digits

    while True:
        stdout.write(choice(chars))
        if count % 80 == 0:
            stdout.write('\n')
        stdout.flush()
        sleep(0.01)

        count += 1

File: kung_fury.rb
require "securerandom"

if __FILE__ == $0
  begin
    f = File.open("script.txt", "r")
  rescue Errno::ENOENT
    puts "Could not open file"
    exit
  end

  energy_converter_text = f.read
  (0...2).each do |i|
    energy_converter_text.each_char do |c|
      print(c)
      sleep(0.02)
    end 
  end

  count = 0
  
  while true
    SecureRandom.hex.each_char do |c|
      print(c)
      print("\n") if count % 80 == 0
      sleep(0.01)

      count += 1
    end
  end
end
